src/app/app.ts

```diff
@@ -16,7 +16,6 @@ import {BrowserModule} from '@angular/platform-browser';
   selector: 'rio-app',
   // Global styles imported in the app component.
   encapsulation: ViewEncapsulation.None,
-  styleUrls: [ '../styles/index.css' ],
```

Comment:
After AoT compilation, styleUrls causes problem and stop angular from rendering component.